app/services/funnel/doc_auth/register_step.rb

```diff
@@ -5,7 +5,6 @@ class RegisterStep
         agreement
         welcome
         upload
-        link_sent
```

Comment:
Do we want to delete `link_sent`? The data isn't used, but it's still being written to via the FSM LinkSentStep. I would wait on this until that step is pulled out of the Flow State Machine. I'm surprised that didn't break any tests.

Reply:
Gotcha - restored in 4b58deaca